pages/common/emacs.md

```diff
@@ -14,3 +14,7 @@
 - Exit emacs (save buffers and terminate):
 
 `Ctrl + X, Ctrl + C`
+
+- Open a file in emacs at a line number:
```

Comment:
Maybe `at a specified line number` would read better?

Reply:
I like that.  I'll make the change.